server/netconn.go

```diff
@@ -0,0 +1,65 @@
+package server
```

Comment:
Created to be able to pass a Request to Thread pool request channelby translating the syscall.Accept response to net Request.